src/web/components/Onwards/Carousel/Carousel.tsx

```diff
@@ -420,7 +289,7 @@ export const Carousel: React.FC<OnwardsType> = ({
 	});
 
 	const cards = trails.map((trail, i) => (
-		<Card trail={trail} isFirst={i === 0} />
+		<CarouselCard key={trail.url + i} trail={trail} />
 	));
```

Comment:
It might save a bit of scrolling around if this was just inlined inside the `ul`. I think now that it is just that `ul` tag there's no more need to extract it out?